bin/setup.sh

```diff
@@ -1,90 +1,104 @@
 
-# <<PHP
-#   define( 'WP_DEBUG', true );
-#   define( 'WP_DEBUG_LOG', true );
-#   define( 'WP_DEBUG_DISPLAY', false );
-#   @ini_set( 'display_errors', 0 );
-#   define( 'SAVEQUERIES', false );
-#   define( 'JETPACK_DEV_DEBUG', true );
-#   define( 'WP_CACHE', true );
-#   define( 'WP_CACHE_KEY_SALT', '$WP_CACHE_KEY_SALT' );
-#   define( 'WP_ENV', 'development' );
-# PHP
-
-  if [ $MULTISITE -eq 1 ]; then
-    echo " * Setting up multisite \"$TITLE\" at $URL"
-    docker-compose exec --user www-data phpfpm wp core multisite-install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL" --subdomains
-    docker-compose exec --user www-data phpfpm wp super-admin add admin
-
-  else
-    echo " * Setting up \"$TITLE\" at $URL"
-    docker-compose exec --user www-data phpfpm wp core install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL"
-  fi
-
-  ## ACTIVATING COMPONENTS ##
-  echo " * Activating the default theme"
-  docker-compose exec --user www-data phpfpm wp theme activate ${THEME}
-
-  # echo " * Importing test content"
-  # docker-compose exec --user www-data phpfpm curl -OLs https://raw.githubusercontent.com/manovotny/wptest/master/wptest.xml
-  # docker-compose exec --user www-data phpfpm wp import wptest.xml --authors=create
-  # docker-compose exec --user www-data phpfpm rm wptest.xml
-fi
+	read -p "is Multisite (y/n)? default n: " is_multisite
+	case "${is_multisite}" in
+		y|Y ) MULTISITE=1;;
+		n|N ) MULTISITE=0;;
+		* ) MULTISITE=0;;
+	esac
+
+	if [ ${MULTISITE} -eq 1 ]; then
+		echo " * Setting up multisite \"${TITLE}\" at ${URL}"
+		docker-compose exec --user www-data phpfpm wp core multisite-install --url="$URL" --title="${TITLE}" --admin_user=admin --admin_password=password --admin_email="${ADMIN_EMAIL}" --subdomains
+		docker-compose exec --user www-data phpfpm wp super-admin add admin
+	else
+		echo " * Setting up \"${TITLE}\" at ${URL}"
+		docker-compose exec --user www-data phpfpm wp core install --url="${URL}" --title="${TITLE}" --admin_user=admin --admin_password=password --admin_email="${ADMIN_EMAIL}"
+	fi
+
+	docker-compose exec --user www-data phpfpm wp db create --url="${URL}" --path="${ROOT}/wordpress"
 
-## UPDATING COMPONENTS ##
-echo "Updating WordPress"
-docker-compose exec --user www-data phpfpm wp core update
-docker-compose exec --user www-data phpfpm wp core update-db
+}
+
+read -p "Install Wordpress core? (y/n)? default y: " is_install
+case "${is_install}" in
+	y|Y ) wordpress_install;;
+	n|N ) break;;
+	* ) wordpress_install;;
+esac
+
+wordpress_updater() {
+	## UPDATING COMPONENTS ##
+	echo "Updating WordPress"
+	docker-compose exec --user www-data phpfpm wp core update
+	docker-compose exec --user www-data phpfpm wp core update-db
+}
+
+if [ "${is_install}" == 'n' ] || [ "${is_install}" == "N" ]; then
```

Comment:
You don't need this validation.  
Even if you install WP core, you can always ask for an update.